### PR TITLE
Portfolio Menu Implementation

### PR DESCRIPTION
**Pull request by cyrilmuchemi**
My partner, @cyrilmuchemi, and I, have added code to the hamburger menu to display other features within the portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -361,5 +361,27 @@ <h2>Get started</h2>
...
+            </header>
+
+            <section class="menu">
+                <div class="content-menu">
+                    <ul class="menu-list">
+                        <li><a class="close-menu-headline" href="#headline"><h4>Hello</h4></a></li>
+                        <li><a class="close-menu-work" href="#works"><h4>Portfolio</h4></a></li>
+                        <li><a class="close-menu-about" href="#about-myself"><h4>About</h4></a></li>
+                        <li><a class="close-menu-contact" href="#contact"><h4>Contact</h4></a></li>
+                    </ul>
```
Comment (prolajumokeoni) [inline index.html:379]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_7 | Screenshot_6


- [ ] Kindly reduce spacing  before and after the nav links, so it looks like the Figma design

File: index.html [hunk 2]
```diff
@@ -361,5 +361,27 @@ <h2>Get started</h2>
                 </footer>
             </section>
         </div>
+
+        <div class="wrapper-menu" id="wrapper-menu">
+            <header class="header">
+                <nav class="nav">
+                    <a class="close-menu" href="#"><img class="menu-button" src="img/Union.svg" alt="Menu Buttom"></a>
```
Comment (prolajumokeoni) [inline index.html:368]: - [ ] Kindly add more padding or margin top  to the close button, check screenshot below ⬇️

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi Team 👋🏿 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ GitHub flow was correctly used
✔️  Good PR and Description



## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @team , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**